doc/sphinx/source/buildlb.rst

```diff
@@ -180,6 +161,58 @@ The process documented here will work on CentOS 7.4.
       $ make && sudo make install 
 
 
+.. _ngraph_plaidml_backend:
+
+Building nGraph-PlaidML from source
+===================================
+
+The following instructions will create the ``~/ngraph_plaidml_dist`` 
+locally:
+
+#. Ensure you have installed the :ref:`prerequisites` for your OS.
+
+#. Install the prerequisites for the backend. Our hybrid ``NGRAPH_PLAIDML``
+   backend works best with Python3 versions. Feel free to use a virtual
```

Comment:
"Feel free to use a virtual environment" should probably be "We recommend that you use a virtual environment" due to some of the difficulties that users have seen when trying to install outside of a venv.

Reply:
Done